plover/gui/config.py

```diff
@@ -200,20 +202,30 @@ def __init__(self, config, parent):
         sizer = wx.FlexGridSizer(2, 3)
         sizer.AddGrowableCol(1)
 
-        sizer_flags = wx.SizerFlags(1).Align(wx.ALIGN_CENTER_VERTICAL).Border(wx.ALL, UI_BORDER)
+        sizer_flags = wx.SizerFlags(1) \
+            .Align(wx.ALIGN_CENTER_VERTICAL) \
+            .Border(wx.ALL, UI_BORDER)
 
-        sizer.AddF(wx.StaticText(self, label=MACHINE_LABEL), sizer_flags.Left())
+        sizer.AddF(
+            wx.StaticText(self, label=MACHINE_LABEL),
+            sizer_flags.Left())
         machines = machine_registry.get_all_names()
         current_machine = self.config.get_machine_type()
         self.choice = wx.Choice(self, choices=machines)
-        self.choice.SetStringSelection(machine_registry.resolve_alias(current_machine))
+        selected_machine = machine_registry.resolve_alias(current_machine)
+        self.choice.SetStringSelection(selected_machine)
         sizer.AddF(self.choice, sizer_flags.Expand())
         self.Bind(wx.EVT_CHOICE, self._update, self.choice)
-        self.config_button = wx.Button(self, id=wx.ID_PREFERENCES, label=CONFIG_BUTTON_NAME)
+
+        self.config_button = wx.Button(
+            self,
+            id=wx.ID_PREFERENCES,
+            label=CONFIG_BUTTON_NAME)
         sizer.AddF(self.config_button, sizer_flags.Right())
         self.Bind(wx.EVT_BUTTON, self._advanced_config, self.config_button)
 
-        self.auto_start_checkbox = wx.CheckBox(self, label=MACHINE_AUTO_START_LABEL)
+        self.auto_start_checkbox = wx.CheckBox(
```

Comment:
So flake8 insists that, if 1 param is on the same line as the open paren, then all must be at least that indented. But if you kick all the params off the line with the open paren, then any indentation is kosher by it. I mostly prefer to keep things near the left edge, so I preferred to evict all params from the line when wrapping was required due to line length.